tools/depends/native/TexturePacker/src/decoder/IDecoder.h

```diff
@@ -70,3 +71,25 @@ class DecodedFrames
     DecodedFrames() = default;
     std::vector<DecodedFrame> frameList;
 };
+
+template<>
+struct std::hash<DecodedFrames>
+{
+    std::size_t operator()(DecodedFrames const& frames) const noexcept
+    {
+      std::size_t seed = frames.frameList.front().rgbaImage.pixels.size();
+
+      for (const auto& frame : frames.frameList)
+      {
+        for (auto x : frame.rgbaImage.pixels)
+        {
+          x = ((x >> 16) ^ x) * 0x45d9f3b;
+          x = ((x >> 16) ^ x) * 0x45d9f3b;
+          x = (x >> 16) ^ x;
+          seed ^= x + 0x9e3779b9 + (seed << 6) + (seed >> 2);
+        }
```

Comment:
```suggestion
        auto frameHash = std::hash<std::string_view>()(
            std::string_view(reinterpret_cast<const char*>(frame.rgbaImage.pixels.data()),
                             frame.rgbaImage.pixels.size()));
        seed ^= frameHash + 0x9e3779b9 + (seed << 6) + (seed >> 2);
```
Not an expert on hash functions, but it looks to me as this one is intended for integers? My suggestion is to use the std::string_view hasher, I assume it will give good results.